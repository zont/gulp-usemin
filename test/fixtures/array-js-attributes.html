<!--build:js /dist/index.js -->
<script src="js/lib.js"></script>
<!--endbuild-->

<!--build:js1 /dist/index1.js -->
<script src="js/main.js"></script>
<!--endbuild-->

<!--build:js2 /dist/index2.js -->
<script src="js2/lib2.js"></script>
<!--endbuild-->

<!--build:js3 /dist/index3.js -->
<script src="js2/main2.js"></script>
<!--endbuild-->