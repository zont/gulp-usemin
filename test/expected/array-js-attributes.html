<script src="/dist/index.js" seq="1" color="blue"></script>

<script src="/dist/index1.js" seq="2" color="red"></script>

<script src="/dist/index2.js" seq="1" color="yellow"></script>

<script src="/dist/index3.js" seq="3" color="pink"></script>